src/focus.service.ts

```diff
@@ -398,7 +392,6 @@ export class FocusService {
       // Trigger focus changes and add selected classes everywhere
       // from the target element to the root.
       for (let el = next; el !== this.root && el; el = el.parentElement) {
-        el.classList.add(cssClass.selected);
         this.triggerFocusChange(el, next);
```

Comment:
This is pretty much free, actually. Running the benchmark, triggerFocusChange didn't even take a tenth of a millisecond.

![](https://peet.io/i/17-03-toozb4hkqe7ub0t056c5004jn3rkh5.png)